packages/makeup-roving-tabindex/docs/bundled/docs.min.js

```diff
@@ -60,22 +60,22 @@
       })()
         ? n
         : "undefined" != typeof document && "function" == typeof document.createEvent
-        ? function (e, t) {
-            var n = document.createEvent("CustomEvent");
-            return (
-              t ? n.initCustomEvent(e, t.bubbles, t.cancelable, t.detail) : n.initCustomEvent(e, !1, !1, void 0), n
-            );
-          }
-        : function (e, t) {
-            var n = document.createEventObject();
-            return (
-              (n.type = e),
-              t
-                ? ((n.bubbles = Boolean(t.bubbles)), (n.cancelable = Boolean(t.cancelable)), (n.detail = t.detail))
-                : ((n.bubbles = !1), (n.cancelable = !1), (n.detail = void 0)),
-              n
-            );
-          };
+          ? function (e, t) {
```

Comment:
May want to ignore this generated/minified file also?

Reply:
I think I'd wanna ignore bundled/built files for sure.